pandas/core/groupby/ops.py

```diff
@@ -198,13 +198,10 @@ def apply(self, f: F, data: FrameOrSeries, axis: int = 0):
             try:
                 result_values, mutated = splitter.fast_apply(f, sdata, group_keys)
 
-            except libreduction.InvalidApply as err:
-                # This Exception is raised if `f` triggers an exception
-                # but it is preferable to raise the exception in Python.
-                if "Let this error raise above us" not in str(err):
-                    # TODO: can we infer anything about whether this is
-                    #  worth-retrying in pure-python?
-                    raise
+            except IndexError:
+                # test_apply_mutate this is a rare case in which re-running
```

Comment:
which test is this referring to? (I don't see a `test_apply_mutate`)

Reply:
test_apply_mutate.py, will update to make clearer

Reply:
@jbrockmendel is there something you were going to update here?

Reply:
thanks for the reminder, just updated